lib/tasks/parallel_tests.rake

```diff
@@ -0,0 +1,6 @@
+desc "Run specs in parallel"
+task :parallel, [:cores] do |_task, args|
```

Comment:
Is it worth documenting this one in the REAMDE?

Reply:
I've removed this - ended up being unnecessary 😁